model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -209,15 +209,15 @@ class FindingsMatcherTest : WordSpec() {
         }
 
         "Given a file with multiple license and a not nearby copyright finding and a root license, match" should {
-            "discard that statement" {
+            "associated that finding with the root license" {
                 setupLicenseFinding("license 1", "path", 1)
                 setupLicenseFinding("license 2", "path", 2)
                 setupCopyrightFinding("statement 1", "path", 2 + DEFAULT_TOLERANCE_LINES + 1)
                 setupLicenseFinding("root license 1", NESTED_LICENSE_FILE_A, 102)
 
                 val result = matcher.match(licenseFindings, copyrightFindings)
 
-                result.getAllStatements() should beEmpty()
+                result.getFindings("root license 1").copyrights.map { it.statement } shouldBe listOf("statement 1")
```

Comment:
`should containExactly` instead?

Reply:
I prefer to keep it for now for (local) consistency. I'll make a follow-up PR to fix it for the whole file.